_vale/config/vocabularies/Docker/accept.txt

```diff
@@ -1,10 +1,11 @@
-(?-i)[A-Z]{2,}s
+(?-i)[A-Z]{2,}'?s
 Amazon
 Anchore
 Apple
 Artifactory
 Autotest
 Azure
+Berkely
```

Comment:
Actually this is a typo, shouldn't be in the vocabulary. The correct spelling is Berkeley